frontend/pages/DashboardPage/cards/ActivityFeed/GlobalActivityItem/GlobalActivityItem.tsx

```diff
@@ -1372,6 +1372,9 @@ const GlobalActivityItem = ({
         ) : (
           DEFAULT_ACTOR_DISPLAY
         );
+      case ActivityType.UserFailedLogin:
+      case ActivityType.FleetEnrolled:
+        return "";
```

Comment:
I'm a bit worried there are missing activities cases here, maybe a better fix would be on line 1355 to set `DEFAULT_ACTOR_DISPLAY` to `Fleet` only if the activity is `fleet_initiated`? So that it stays empty for other cases?

Reply:
Are we pretty confident at this point that all of the "Fleet" activities have the `fleet_initiated` flag implemented on the backend now? If so, I definitely think what you're suggesting makes the most sense. 

If not, I guess the question boils down to picking the least worst alternative between "Fleet told Fleet" and "_ told Fleet". I could go either way, but I'm slightly inclined toward leaving it empty in cases where it is not explicitly fleet initiated as you are suggesting as it forces us to make it right on the backend instead of relying on a UI fallback.

Reply:
> Are we pretty confident at this point that all of the "Fleet" activities have the fleet_initiated flag implemented on the backend now?

It is set for all script executions and software installs triggered by a failed policy. The only problem, I think, is that existing activities would not have the proper formatting without the frontend assistance.

> I'm slightly inclined toward leaving it empty in cases where it is not explicitly fleet initiated as you are suggesting as it forces us to make it right on the backend instead of relying on a UI fallback

I agree with that rationale. I think I'll go ahead with this change.